elastic/kibana
# [Ingest Manager] During fleet setup create an enrollment for every agent config

## Description

Resolve #71266 

During setup we were only creating an enrollment token for the default config not for the other configs. This PR fix it.


## How to test this?
From a new ES and kibana environement:
*  create an agent config, 
* setup fleet 
* go to the enrollment token list (in the fleet tab) you should see two enrollment token one for the default config, one for the one you created.
